client/state/sites/hooks/use-site-global-styles-status.ts

```diff
@@ -67,14 +51,11 @@ const getGlobalStylesInfoForSite = ( siteId: number | null ): Promise< GlobalSty
 	}
 
 	if ( siteId === null ) {
-		return loadExperimentAssignment( 'calypso_global_styles_personal_v2' ).then(
-			( experimentAssignment ) =>
-				Promise.resolve( {
-					shouldLimitGlobalStyles: true,
-					globalStylesInUse: false,
-					globalStylesInPersonalPlan: experimentAssignment.variationName === 'treatment',
-				} )
-		);
+		return Promise.resolve( {
+			shouldLimitGlobalStyles: true,
+			globalStylesInUse: false,
+			globalStylesInPersonalPlan: false,
+		} );
```

Comment:
From the check in line 45 we can see that this code will never be reached if the `Premium plan` variation wins. I think it's safe to replace the contents of this function with the promise resolution being highlighted by this comment.

Reply:
But anyway, this shouldn't be a blocker, so feel free to move forward with this.